rclcpp/src/rclcpp/executors/multi_threaded_executor.cpp

```diff
@@ -51,6 +51,13 @@ MultiThreadedExecutor::~MultiThreadedExecutor() {}
 
 void
 MultiThreadedExecutor::spin()
+{
+  spin(std::function<void(const std::exception &)>());
+}
+
+void
+
+MultiThreadedExecutor::spin(std::function<void(const std::exception & e)> exception_handler)
```

Comment:
how about adding the test case for this to make sure the callback suppresses the exception?

Reply:
Added test case